lib/beaker/logger.rb

```diff
@@ -83,6 +83,25 @@ def initialize(*args)
 
       @destinations = []
 
+      @color_levels = options[:color_levels] || {
+        :error => RED,
+        :warn => BRIGHT_RED,
+        :success => MAGENTA,
+        :notify => BLUE,
+        :info => GREEN,
+        :debug => WHITE,
+        :trace => BRIGHT_YELLOW,
+        :perf => BRIGHT_MAGENTA,
+        :host => YELLOW
+      }
+
+      # Jenkins exposed variable - should be present on the slave directing
+      # the beaker run
+      ci_build = ENV['BUILD_NUMBER'] != nil
+
+      @color_levels[:notify] = NORMAL if ci_build
```

Comment:
I thought about this. I think right now it's definitely what is wanted. Perhaps we add an additional option not to allow CI to override colors?